test/preflight/testlib/test_env.go

```diff
@@ -180,6 +181,10 @@ type testingTWrapper interface {
 
 // Fly runs a flyctl the result
 func (f *FlyctlTestEnv) Fly(flyctlCmd string, vals ...interface{}) *FlyctlResult {
+	if strings.HasPrefix(flyctlCmd, "machine run ") || strings.HasPrefix(flyctlCmd, "launch ") {
+		flyctlCmd += " --vm-gpu-kind a100-sxm4-80gb --vm-cpu-kind performance --vm-memory 8192 "
```

Comment:
A few notes:

* `a100-sxm4-80gb` are scarse
* `a100-pcie-40gb` or `l40s` availability are more common but only on `iad` and `ord` regions
* `--vm-size a100-40gb` or `--vm-size l40s` are useful shortcuts